src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -162,15 +163,20 @@ static MvcCoreLoggerExtensions()
                 1,
                 "Executing ContentResult with HTTP Response ContentType of {ContentType}");
 
-            _actionMethodExecuting = LoggerMessage.Define<string, string[], ModelValidationState>(
+            _actionMethodExecuting = LoggerMessage.Define<string, ModelValidationState>(
+                LogLevel.Information,
+                0,
+                "Executing action method {ActionName} - {ValidationState}");
+
+            _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[], ModelValidationState>(
                 LogLevel.Information,
                 1,
-                "Executing action method {ActionName} with arguments ({Arguments}) - ModelState is {ValidationState}");
+                "Executing action method {ActionName} with arguments ({Arguments}) - {ValidationState}");
```

Comment:
So it looks like this one _actionMethodExecuting got split into two? They should have the same log ID.